app/ledger.js

```diff
@@ -838,7 +838,10 @@ var run = (delayTime) => {
     delayTime = client.timeUntilReconcile()
     if (delayTime === false) delayTime = 0
   }
-  if (delayTime > 0) return setTimeout(() => { if (client.sync(callback) === true) return run(0) }, delayTime)
+  if (delayTime > 0) {
+    active = client
```

Comment:
the original bug was caused by going from 'on -> off' and then having the timer fire.

if someone does 'on -> off -> on', then client has changed, and when the timer fires, the old client is used..

that's why there's a check that the client instance hasn't changed.  your thinking?